tests/test_split.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import numpy as np
+from mvlearn.model_selection import train_test_split
+
+RANDOM_SEED = 10
+np.random.seed(RANDOM_SEED)
```

Comment:
never use global seed that have effect across tests but use np.random.RandomState